client/webserver/site/src/js/wallets.ts

```diff
@@ -411,6 +411,7 @@ export default class WalletsPage extends BasePage {
     const page = this.page
     const markets: [string, Market][] = []
     for (const xc of Object.values(app().user.exchanges)) {
+      if (!xc.markets) continue
```

Comment:
Just confirming that 0.5 has this check (twice, redundantly, actually) in it's `showMarkets`.

Reply:
`setSelectedAsset` will run when the wallets page loads hence the error from `showAvailableMarkets`

Reply:
I understand.  I'm saying the bug is only on master (does not need backport for release).